templates/web/common/site_layout.tt.html

```diff
@@ -20,7 +20,7 @@
     <link rel="stylesheet" href="[% static_subdomain %]/css/dist/jqueryui/themes/base/jquery-ui.css">
     <link rel="stylesheet" href="[% static_subdomain %]/css/dist/select2.min.css">
     <link rel="search" href="[% formatted_subdomain %]/cgi/opensearch.pl" type="application/opensearchdescription+xml" title="[% lang('site_name') %]">
-	<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+	<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Public+Sans&family=Material+Icons">
```

Comment:
Don't you want to host it on the server ?

Reply:
@alexgarel it's in a different issue: https://github.com/openfoodfacts/openfoodfacts-server/issues/7474